File: backend/Rest API PWII/Models/ViewModels/SearchResultUserModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Rest_API_PWII.Models.ViewModels
{
    public class SearchResultUserModel
    {
        public string   UserId          { get; set; }
        public string   UserName        { get; set; }
        public string   UserTag         { get; set; }
        public int?     FollowerCount   { get; set; }
        public string   ProfilePicPath  { get; set; }
    }
}

File: frontend/src/model/SearchResultUserModel.js
class SearchResultUserModel{
    userName;
    tag;
    followerCount;
    profilePicID;

    constructor({userName,tag,followerCount,profilePicID}){
        this.userName       = userName;
        this.tag            = tag;
        this.followerCount  = followerCount;
        this.profilePicID   = profilePicID;
    }
}

export default SearchResultUserModel;